onnxruntime/test/contrib_ops/attention_op_test_helper.h

```diff
@@ -30,14 +30,16 @@ struct AttentionTestData {
   std::vector<float> bias_data;
   std::vector<float> rel_pos_bias_data;
 
-  std::vector<float> past_key_data;
-  std::vector<float> past_value_data;
-
-  std::vector<float> fp32_output_data;
-  std::vector<float> fp16_output_data;
-
-  std::vector<float> present_key_data;
-  std::vector<float> present_value_data;
+  struct OutputData {
+    std::vector<float> past_key_data;
```

Comment:
Past in input. It shall not be impacted by bias. Shall we move it out of OutputData?

Reply:
if present is impacted, then past will be. Because from the point of decoding, in the next step, the previous `present` will be the current `past`.

Reply:
struct unchanged. No need to use `OutputData` anymore.